clients/java/signalr/src/main/java/com/microsoft/signalr/HubException.java

```diff
@@ -3,16 +3,32 @@
 
 package com.microsoft.signalr;
 
+/**
+ * The exception thrown from a hub when an error occurs.
```

Comment:
Suggest: An exception thrown when the server fails to invoke a Hub method.

Reply:
Yuck. Don't like it there either :). We can fix it there later, but let's not perpetuate it unless we think it's clearer.

Reply:
Cool